### PR TITLE
For easily using jsonnet in test.

### PR DESCRIPTION
**Pull request by lamuguo**
My use case is: based on the built image, I can add maven and corresponding Java staff into it, and then I can use the command line to generate json / protobuf for config files, like gcl did.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:latest
+
+WORKDIR /opt
+
+RUN apt-get update && \
+    apt-get install -y git build-essential autoconf libtool make
+
+# Install protoc
```
Comment (mikedanese) [inline Dockerfile]: protoc?

Comment (lamuguo) [inline Dockerfile]: Done.

**Comment by lamuguo** [comment]
btw, I updated the CLA info, would like to know how to trigger another CLA check. Thanks!

**Comment by sparkprime** [comment]
I manually verified you signed the CLA, so please rebase and I'll merge.

**Comment by lamuguo** [comment]
Hi @sparkprime ,

I already rebased, but looks nothing happened. Anything I should do to disable "cla:no"? Thanks!

Best Regards,
Xiaofeng

**Comment by sparkprime** [comment]
Sorry I meant squash to one commit as there's no point having that intermediate state in the core repo :)  That might make the cla:no go away because it will trigger the bot to re-check but if not I can merge it anyway as I manually validated your CLA status.

**Comment by lamuguo** [comment]
Looks it works. Thanks, @sparkprime !

Would you take another look?

**Pull request merged**